File: src/test/python/sample/CountLettersEvaluatorTest.py
import unittest
from sample.CountLettersEvaluator import CountLettersEvaluator


class TestCountLettersEvaluator(unittest.TestCase):

    def test_evaluate_hello(self):
        self.assertEqual(CountLettersEvaluator.evaluate(
            "hello"), 5, "when string is 'hello', then returns  5")

    def test_evaluate_empty(self):
        self.assertEqual(CountLettersEvaluator.evaluate(
            ""), 0, "when string is empty, then returns  0")


if __name__ == '__main__':
    unittest.main()

File: src/test/kotlin/sample/CountLettersEvaluatorTest.kt
package sample

import org.junit.Assert.assertEquals
import org.junit.Test
import org.junit.runner.RunWith
import org.junit.runners.JUnit4

@RunWith(JUnit4::class)
class CountLettersEvaluatorTest {

  private val evaluator = CountLettersEvaluator()

  @Test
  fun `when string is "hello", then returns  5`() {
    val result = evaluator.evaluate("hello")
    assertEquals(5, result)
  }

  @Test
  fun `when string is empty, then returns  0`() {
    val result = evaluator.evaluate("")
    assertEquals(0, result)
  }
}
